uploader/api_commands.py

```diff
@@ -55,14 +56,21 @@ def upload_video():
     processed_video = processed_video.resize(height=720)  # Resize height to 720p
     processed_video = processed_video.crop(x_center=processed_video.w / 2, y_center=processed_video.h / 2, width=1280,
                                  height=720)  # Crop to 16:9
-
     # Save the final video
-    # TODO: We need to update the DB register to status 'processed' and update the file url with the processed file
     processed_video.write_videofile(
         os.path.join(current_processed_folder, 'processed_' + filename_with_timestamp))
 
-    return 'File uploaded and processed successfully'
-
+    # TODO: We need to update the DB register to status 'processed' and update the file url with the processed file
+    video_to_be_updated = Video.query.filter(Video.id == video.id).first()
+    if video_to_be_updated is not None:
+        video_to_be_updated.processed_file_url = os.path.join(current_processed_folder, 'processed_' + filename_with_timestamp)
+        video_to_be_updated.status = Status.processed
+        db.session.commit()
+
+    return jsonify(
+        id= video_to_be_updated.id,
+        message='File uploaded and processed successfully'
```

Comment:
Cuando se implemente celery, solo deberia ser 
```suggestion
        message='File uploaded successfully'
```